package/linux/make-package

```diff
@@ -307,5 +307,30 @@ if [ "${RSTUDIO_BUILD_PACKAGE:-1}" = "1" ]; then
    cpack --verbose --debug -G "$PACKAGE_TARGET"
 fi
 
+# For Jenkins builds, upload debug symbols to S3.
+upload-debug-symbols () {
+
+   # Validate that the requisite environment variables are defined.
+   if ! all-defined JENKINS_URL PRODUCT OS ARCH; then
+      return 1
+   fi
+
+   # Don't upload debug symbols for builds triggered by pull requests.
+   case "${WORKSPACE}" in
```

Comment:
Do we also need to avoid locally built packages? Oh, or does the JENKINS_URL check above protect us from that?